IS7024FinalProjectBreweryArc/BreweryArc/Pages/Index.cshtml

```diff
@@ -12,7 +12,7 @@
 
 <a href="/" title="home" class="logo">
     <div class="logo">
-        <img src="~/images/logo.jpg" width="200" alt="logo" />
+        <img src="~/images/logo.jpg" class="imagestyle" alt="logo" />

```

Comment:
Added a new class for image style.

Reply:
Yes.  Linked style is better than inline.  Inline is not permitted in XHTML 1.0 Strict.